PennyMe/ArtworkViews.swift

```diff
@@ -15,25 +15,29 @@ class ArtworkMarkerView: MKMarkerAnnotationView {
     override var annotation: MKAnnotation? {
         
         willSet {
+            // define subtitle subtitles
+            let detailLabel = UILabel()
+            detailLabel.numberOfLines = 0
+            detailLabel.font = detailLabel.font.withSize(12)
+            
             // 1
             let check = newValue?.title
             if check == "New Machine"{
                 guard let newmachine = newValue as? NewMachine else {
                     return
                 }
-                // Create view when marker is pressed
-                let identifier = "markerNewMachine"
+                // define annotation view
                 var view: MKMarkerAnnotationView
+                let identifier = "newmachine"
                 view = MKMarkerAnnotationView(annotation: annotation, reuseIdentifier: identifier)
                 view.canShowCallout = true
-                view.calloutOffset = CGPoint(x: 0, y: 0)
-                // Multiline subtitles
-                let detailLabel = UILabel()
-                detailLabel.numberOfLines = 0
-                detailLabel.font = detailLabel.font.withSize(12)
+                view.calloutOffset = CGPoint(x: -5, y: 5)
+                
+                // add callouts (only address as subtitle)
                 detailLabel.text = newmachine.text
                 detailCalloutAccessoryView = detailLabel
                 rightCalloutAccessoryView = nil
+                leftCalloutAccessoryView = nil
```

Comment:
with this, it at least shouldn't happen at all for new machines, which is already some progress